src/test/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/integration/SoftDeleteJobTest.kt

```diff
@@ -69,6 +74,14 @@ class SoftDeleteJobTest : SqsIntegrationTestBase() {
     // We don't re-delete previously deleted licences
     assertThat(versions[16]).isEqualTo(LocalDateTime.of(2022, 7, 27, 15, 0, 0, 0))
     assertThat(versions[11]!!.toLocalDate()).isEqualTo(LocalDate.now())
+
+    val events = auditEventRepository.findAll()
+    assertThat(events).hasSize(2)
+
+    assertThat(events).extracting("user", "action", "details").containsExactly(
+      Assertions.tuple("SYSTEM_JOB", "RESET", mapOf("bookingId" to 10)),
```

Comment:
**quibble:** would usually statically import things like this so you don't need to add `Assertions.`